lib/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart

```diff
@@ -47,158 +51,74 @@ class _FindAddressWithReverseGeocodeSampleState
   Widget build(BuildContext context) {
     return Scaffold(
       key: _scaffoldStateKey,
-      endDrawer: Drawer(
-        child: SafeArea(
-          child: Padding(
-            padding: const EdgeInsets.all(10),
-            child: buildServiceControls(context),
-          ),
-        ),
-      ),
       resizeToAvoidBottomInset: false,
```

Comment:
```suggestion
```
I think we can remove this now too, as it was used to position the keyboard when we had search functionality.